tests/Unit/ControlSystem/Test_PiecewisePolynomial.cpp

```diff
@@ -1,101 +1,79 @@
 // Distributed under the MIT License.
 // See LICENSE.txt for details.
 
-#include "tests/Unit/TestingFramework.hpp"
```

Comment:
Per our guidelines, this include has to come first. Why is it being moved?

Reply:
Ugh... I think this is a bug in clang format 9...

Reply:
Oh, it's a feature where they changed the default behavior or something... I've pushed a new commit that maintains the behavior we've had so far.